cypress/cdn.test.ts

```diff
@@ -2,31 +2,43 @@ import { registerLoginHandler, startTestBox } from "../src/index";
 
 declare global {
   interface Window {
-    test_fakeLoginHandler: (props) => Promise<string | boolean>;
-    test_fakeNavigateHandler: (data) => void;
+    test_fakeLoginHandler: (props: any) => Promise<string | boolean>;
+    test_fakeNavigateHandler: (data: any) => void;
     test_startTestBox: any;
     test_registerLoginHandler: any;
     test_baseTbxConfig: {
       targetOrigin: string;
       allowFullStory: boolean;
-    }
+    };
   }
 }
 
-async function fakeLoginHandler(props) {
+async function fakeLoginHandler(props: any): Promise<string | boolean> {
   console.log("Fake login handler called with", props);
-  return true
+  return true;
 }
 
-function fakeNavigateHandler(data) {
-  return true
+async function fakeNavigateHandler(data: any): Promise<void | boolean> {
+  console.log("Fake navigate handler called with", data);
+  return true;
 }
 
 window.test_fakeLoginHandler = fakeLoginHandler;
-window.test_fakeNavigateHandler = fakeNavigateHandler
-window.test_startTestBox = startTestBox
-window.test_registerLoginHandler = registerLoginHandler
+window.test_fakeNavigateHandler = fakeNavigateHandler;
+window.test_startTestBox = startTestBox;
+window.test_registerLoginHandler = registerLoginHandler;
 window.test_baseTbxConfig = {
   targetOrigin: "localhost",
-  allowFullStory: true
-}
\ No newline at end of file
+  allowFullStory: true,
+};
+
+// tbxConfig
+const config = {
+  navigateHandler: window.test_fakeNavigateHandler,
+  loginHandler: window.test_fakeLoginHandler,
+  logLevel: "error",
+  ...window.test_baseTbxConfig,
+};
+
+// @ts-ignore
+// startTestBox(config);
```

Comment:
Can we remove this?

Reply:
I could, I left there to facilitate testing.